lib/embedded_ansible.rb

```diff
@@ -153,4 +152,9 @@ def self.generate_password
     SecureRandom.base64(18).tr("+/", "-_")
   end
   private_class_method :generate_password
+
+  def self.connection
+    ActiveRecord::Base.connection
```

Comment:
Yeah, I was more whining that ActiveRecord doesn't expose a nice API for doing this outside of making a db connection.